classes/Language.php

```diff
@@ -1109,6 +1111,38 @@ public static function installSfLanguagePack($locale, &$errors = array())
         }
     }
 
+    /**
+     * This method has been introduce in Language out of conveniency but, if you can, prefer
+     * using the GenerateMailTemplatesService or MailTemplateGenerator services.
+     *
+     * @param array $lang_pack
+     * @param array $errors
+     */
+    public static function generateEmailsLanguagePack($lang_pack, &$errors = array())
+    {
+        $locale = $lang_pack['locale'];
+        $sfContainer = SymfonyContainer::getInstance();
+        if (null === $sfContainer) {
+            $errors[] = 'Could not generate emails because container is unavailable';
```

Comment:
Is that a translation key? If yes, it needs to be localized, I guess in _Admin.Notifications.Error_.

Reply:
Ok I changed it, but it's weird all other errors in this file use `Admin.International.Notification`

Reply:
Watch out, it must be written _Cannot blah blah_, not _Can not blah blah_. ;-)

Reply:
Really! I didn't know this writing